service/src/integrationTest/kotlin/fi/espoo/evaka/occupancy/OccupancyTest.kt

```diff
@@ -108,6 +114,32 @@ class OccupancyTest : PureJdbiTest(resetDbBeforeEach = true) {
             it.insertTestDaycareGroup(DevDaycareGroup(id = familyGroup2, daycareId = familyUnitInArea2))
             it.insertTestCaretakers(groupId = familyGroup1, amount = 3.0)
             it.insertTestCaretakers(groupId = familyGroup2, amount = 3.0)
+
+            it.insertTestDaycare(
+                DevDaycare(
+                    id = openingDaycare,
+                    areaId = careArea1,
+                    openingDate = null,
+                    closingDate = today.minusWeeks(1),
+                    providerType = ProviderType.MUNICIPAL,
+                    type = setOf(CareType.CENTRE, CareType.PRESCHOOL, CareType.PREPARATORY_EDUCATION)
+                )
+            )
+            it.insertTestDaycareGroup(DevDaycareGroup(id = openingDaycareGroup, daycareId = openingDaycare))
+            it.insertTestCaretakers(groupId = openingDaycareGroup, amount = 3.0)
+
+            it.insertTestDaycare(
+                DevDaycare(
+                    id = closedDaycare,
```

Comment:
These ids should be the other way around.

Reply:
Fixed